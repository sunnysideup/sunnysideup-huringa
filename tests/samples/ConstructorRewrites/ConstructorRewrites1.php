<?php

class ConstructorRewrites1 {
public function ConstructorRewrites1($foo = null, $bar = null) {}
}