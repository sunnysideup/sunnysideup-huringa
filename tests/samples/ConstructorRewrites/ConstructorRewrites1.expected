<?php

class ConstructorRewrites1 {
public function __construct($foo = null, $bar = null) {}
}
